hops/featurestore_impl/util/fs_utils.py

```diff
@@ -96,53 +96,53 @@ def _get_dataframe_tf_record_schema_json(spark_df, fixed=True):
     example_json = {}
     for col in spark_df.dtypes:
         if col[1] in constants.FEATURE_STORE.TF_RECORD_INT_SPARK_TYPES:
-            example[str(col[0])] = tf.FixedLenFeature([], tf.int64)
+            example[str(col[0])] = tf.io.FixedLenFeature([], tf.int64)
```

Comment:
can you document in the jira that you are also updating this to work with tensorflow 2?

Reply:
absolutely will add to Jira.  tf.io works for 1.15 as well.